internal/store/postgres/grant_repository_test.go

```diff
@@ -127,6 +127,22 @@ func (s *GrantRepositoryTestSuite) TestGetGrantsTotalCount() {
 		s.Nil(actualError)
 	})
 }
+func (s *GrantRepositoryTestSuite) TestListUserRoles() {
+	s.Run("should return roles", func() {
+		expectedRoles := []string{}
+
+		actualResult, actualError := s.repository.ListUserRoles(context.Background(), "user")
+		s.Equal(actualResult, expectedRoles)
+		s.Nil(actualError)
+	})
+
+	s.Run("should return err", func() {
+		_, actualError := s.repository.ListUserRoles(context.Background(), "user")
+
+		s.Nil(actualError)
```

Comment:
the test description tells `should return error` but the expectation is actually expecting nil to the error. We can remove this test case if we're unable to simulate an error